packages/taiko-client/prover/prover.go

```diff
@@ -392,6 +415,34 @@ func (p *Prover) proveOp() error {
 	return iter.Iter()
 }
 
+// aggregateOp aggregates all proofs in buffer.
+func (p *Prover) aggregateOp(tier uint16) error {
+	var wg sync.WaitGroup
```

Comment:
Seems no `wg.Wait()` in this method? And shall we use `errgroup` instead of `wg` here?

Reply:
Done